proposals/0000-linear-types.rst

```diff
@@ -52,7 +52,7 @@ that statically enforce that all I/O happens only on open handles,
 never on closed handles (*no use-after-free*). Moreover, we want such
 API's to enable early closing of handles by the user (*prompt
 deallocation*). Use-after-free and prompt deallocation are hard to
-enforce with current Haskell.
+impossible to enforce with current Haskell.
```

Comment:
Note for later: « hard to impossible to ».

(I don't think that they are impossible though. With a good, unpalatable, application of indexed monad, you can probably enforce such invariants)